src/Uno.UI.Composition/Composition/CompositionAnimation.cs

```diff
@@ -53,20 +53,28 @@ public string Target
 		set => _target = value ?? throw new ArgumentException();
 	}
 
+#if __SKIA__
+	CompositionObject? _currentCompositionObject;
```

Comment:
This prevent re-use of the animation on multiple objects ...

Reply:
Indeed, and the current structure does not allow this at this time, we'll need to refactor this.

Reply:
Can you record some issue to track it please?